docs/conf.py

```diff
@@ -204,3 +207,7 @@ def setup(app):
 
 
 # -- Extension configuration -------------------------------------------------
+
+# Install gym on read the doc
```

Comment:
Perhaps better in a `readthedoc.yaml` file? https://docs.readthedocs.io/en/stable/config-file/v2.html

Reply:
Good point

Reply:
I did not know about it. I tried to change the config online at first but it was making the master branch fail. With the config file, it works ;)